playwright-tests/tests/pots.spec.js

```diff
@@ -48,8 +48,14 @@ test.describe("Admin is logged in", () => {
   });
 });
 
-test("clicking learn more button should...", async ({ page }) => {
-  // TODO:
+test("clicking learn more button should redirect to wtfisqr.com", async ({ page }) => {
+  const newTabPromise = page.waitForEvent("popup");
```

Comment:
Does this "popup" event happen?

It looks like tests are failing "waiting for event "popup""